src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -802,9 +803,12 @@ abstract class CallOp(code: Int, delta: Int, alpha: Int) extends OpCode(code, de
       val gasAdjustment: BigInt = if (validCall) 0 else -startGas
       val mem2 = mem1.expand(outOffset, outSize)
 
+      val updateWorld = if (validCall) state.world.combineTouchedAccounts(result.world) else state.world
```

Comment:
`updatedWorld` tells us nothing, either use the convention and name it `world1` or think of something clever :)